src/content/docs/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-intro.mdx

```diff
@@ -18,20 +27,19 @@ The [OpenTelemetry collector](https://opentelemetry.io/docs/collector/) is a ven
-- [Singlestore](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-singlestore)
-- [Squid](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-squid)
-- [StatsD](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-statsd)
+* [Docker](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/docker)
+* [HCP Consul](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/hcp-consul)
+* [HiveMQ](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/hivemq)
+* [Hosts](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/host-monitoring)
+* [Kafka Confluent Cloud](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/confluentcloud)
+* [Prometheus](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/prometheus)
+* [Redis](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/redis)
+* [Singlestore](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/singlestore)
+* [Squid](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/redis)
+* [StatsD](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/statsd)
 
 ## Correlating infrastructure with OpenTelemetry APM [#infrastructure-correlation]
 
 The following examples demonstrate using the collector to correlate [APM OpenTelemetry data](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro) with infrastructure. The general pattern is to configure the collector with a processor which detects and enriches the APM telemetry with additional environment context in the form of resource attributes, before exporting the data to New Relic via OTLP. New Relic can detect this correlation data and construct [relationships between APM and infrastructure entities](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources).
 
-- [Hosts](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-hosts)
-- [Kubernetes](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-k8s)
+* [Hosts](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-hosts)
```

Comment:
@jack-berg this link is to one of the files that was removed. I am going to remove this link from the doc. If you need to add this back, you can do so later.